### PR TITLE
Updated NxTags to match Kaleido - RSC-718

### PR DESCRIPTION
**Pull request by entroform**
https://issues.sonatype.org/browse/RSC-718

### CODE REVIEW

**Comment by rpokorny** [comment]
I'm getting a dark border on focused selectable tags, which isn't supposed to be there
image

**Comment by entroform** [comment]
> I'm getting a dark border on focused selectable tags, which isn't supposed to be there image

I double checked this with Jenni:
So when it is selected + focused + hovered, there shouldn't be any black border,
but in this case when it is **unselected** + focused + hovered it retains the black border styling from the hover state.

**Comment by rpokorny** [comment]
> I double checked this with Jenni:
So when it is selected + focused + hovered, there shouldn't be any black border,
but in this case when it is unselected + focused + hovered it retains the black border styling from the hover state.
Jenni will update the Kaleido mockup to explicitly specify^

OK but I took the screenshot above when I was merely focused, not also hovering.  Additionally the black border is present in both the selected and unselected focus states.

**Comment by entroform** [comment]
> OK but I took the screenshot above when I was merely focused, not also hovering. Additionally the black border is present in both the selected and unselected focus states.

Ah sorry, for some reason this is only happening in Chrome, I fixed it here:

95b86d66c141dff4869b9e4a09dc3fcbfc7ee69a

File: lib/src/components/NxTag/NxTag.scss [hunk 1]
```diff
@@ -33,25 +33,33 @@
...
 
   &:hover {
     border-color: var(--nx-color-interactive-border-hover);
   }
 
   &:focus {
-    border: 1px solid var(--nx-color-interactive-border-focus);
-    box-shadow: var(--nx-box-shadow-focus);
-    outline: none;
+    &:focus-visible {
```
Comment (rpokorny) [inline lib/src/components/NxTag/NxTag.scss:43]: I'm not under the impression that this extra bit of selector is needed here.  Disabling the native outline works directly on `:focus`

**Review by rpokorny — APPROVED** [review]
+1 except for latest comment

**Pull request merged**